RYUNATOR.lua

```diff
@@ -155,14 +155,23 @@ function get_health(player_num)
     return mem:read_u16(0xFF83EA + (player_offset * player_num))
 end
 
-function get_round_start(player_num)
-    return mem:read_u16(0xFF83EE + (player_offset * player_num))
+function has_control(player_num)
```

Comment:
Now it returns a boolean so I changed the name, returns true as soon as the round starts and players can move.